logger.js

```diff
@@ -1,14 +1,16 @@
 'use strict';
 // Log class
 
+var util      = require('util');
+var tty       = require('tty');
+
 var constants = require('haraka-constants');
+var logfmt = require('logfmt');
```

Comment:
In the interests of keeping things lean, can we move this require into the `_init_format` function, and only load it when that log format has been specified?

Reply:
We use this in `default` to keep support for the older key=val stuff from back when they were in an array or concatenated.

Reply:
Ah, okay.